components/user_settings/notifications/desktop_notification_settings.jsx

```diff
@@ -49,6 +79,32 @@ export default class DesktopNotificationSettings extends React.PureComponent {
                 soundRadio[0] = true;
             }
 
+            if (this.props.sound === 'true') {
+                const sounds = Array.from(Utils.notificationSounds.keys());
+                const options = sounds.map((sound) => {
+                    return {value: sound, label: sound};
+                });
+
+                let allowNotificationSelection = true;
+                if (isDesktopApp() && window.desktop && semver.lte(window.desktop.version, '4.6.0')) {
+                    allowNotificationSelection = false;
+                }
+
+                if (allowNotificationSelection) {
+                    notificationSelection = (<div className='pt-2'>
+                        <ReactSelect
+                            className='notification-sound-dropdown'
+                            id='displaySoundNotification'
+                            options={options}
+                            clearable={false}
+                            onChange={this.setDesktopNotificationSound}
+                            value={this.state.selectedOption}
+                            isSearchable={false}
+                            ref={this.dropdownSoundRef}
+                        /></div>);
+                }
```

Comment:
@esethna should we have a message here explaining that user should use desktop > 4.6 to enable custom sounds instead of not showing anything? this way users don't get confused trying to find the option and it would probably help us have people upgrade their desktop versions.

Reply:
Discussed and not needed